lib/std/private/shlexutils.nim

```diff
@@ -0,0 +1,83 @@
+type State = enum
```

Comment:
If this is a translation of Python's *shlex* module, it should at least have a comment stating so - if only for attribution's sake (even if it is a private module).

Reply:
> If this is a translation of Python's shlex module

maybe look at the python sources before commenting?

> And unless there's already a shlex.nim somewhere in the standard library, the "utils" suffix should be removed.

This is a private module, aka implementation detail